src/sentry/tasks/unmerge.py

```diff
@@ -35,9 +36,9 @@
 
 
 def cache(function):
-    results = {}
+    results: dict[Any, Any] = {}
 
-    def fetch(*key):
+    def fetch(*key: Any) -> Any:
```

Comment:
same here -- don't bother adding `: Any` or `-> Any` since that's harder to fix later and just the same as leaving the parameter types off